src/ActivitylogServiceProvider.php

```diff
@@ -28,6 +28,10 @@ public function register()
 
         $this->app->bind(ActivityLogger::class);
 
+        $this->app->singleton(ActivityLoggerBatch::class);
```

Comment:
I think that we could/should switch to the spatie package tools here.
https://github.com/spatie/laravel-package-tools